ProcessLib/GroundwaterFlowProcess.h

```diff
@@ -179,9 +203,9 @@ class GroundwaterFlowProcess : public Process
     }
 
 private:
-    ProcessVariable const* _hydraulic_head = nullptr;
+    ProcessVariable* _hydraulic_head = nullptr;
 
-    double const _hydraulic_conductivity = 1e-6;
+    double const _hydraulic_conductivity = 1;
```

Comment:
The variable is `const`. Is it intended that the value can not be changed later on?